cli/tests/unit/test_marathon_pod.py

```diff
@@ -0,0 +1,21 @@
+import dcoscli.marathon.main as main
+from dcos import marathon
+
+from mock import create_autospec
+
+
+def test_add_invoked_successfully():
+    _assert_add_invoked_successfully(pod_file_json={"arbitrary": "json"})
+    _assert_add_invoked_successfully(pod_file_json=["more", "json"])
+
+
+def _assert_add_invoked_successfully(pod_file_json):
+    pod_file_path = "some/path/to/pod.json"
+    resource_reader = {pod_file_path: pod_file_json}.__getitem__
+    marathon_client = create_autospec(marathon.Client)
+
+    pod = main.MarathonPodSubcommand(resource_reader, marathon_client)
+    returncode = pod.add(pod_file_path)
+
+    assert returncode == 0
+    marathon_client.add_pod.assert_called_with(pod_file_json)
```

Comment:
Sure, I'll add one test case that shows an exception raised by `marathon.Client.add_pod()` is also raised by `MarathonPodSubcommand.add()`. That will be explicit about the error behavior.

The reason I didn't already have that test case is that it's implied by:

1. The fact that `MarathonPodSubcommand.add()` calls `marathon.Client.add_pod()` (demonstrated by the above test cases)
1. The assumption that the code being tested is as simple as possible, while still passing all tests. This implies that any exception raised by `marathon.Client.add_pod()` will not be caught in `MarathonPodSubcommand.add()`; otherwise there would have to be a test case demonstrating that behavior in order to justify the existence of a `try...except` block in the code.

But yeah, usually it's better to be explicit, so I'll add the test :smile: